docs/deployment-examples/sql-server-databases.md

```diff
@@ -1,48 +1,91 @@
+
+[Redgate's DLM Automation Suite](https://www.red-gate.com/products/dlm/dlm-automation/index), [Microsoft's DacPac](https://docs.microsoft.com/en-us/sql/relational-databases/data-tier-applications/data-tier-applications?view=sql-server-2017), and [Microsoft's Entity Framework Migrations](https://msdn.microsoft.com/en-us/library/jj591621(v=vs.113).aspx) use the model-driven approach.
+
+2. A change-driven approach, where schema changes are manually written and only run once.  The target database keeps track of which scripts already ran.
+
+[Redgate's ReadyRoll](https://www.red-gate.com/products/sql-development/readyroll/index), [DbUp](https://dbup.readthedocs.io/en/latest/), and [RoundhousE](http://projectroundhouse.org/) use the change-driven approach.
+
+There are pros and cons to either approach as well as the tools themselves.  It is important for your team to research the tools and determine the best one for you.
+
+## Tentacle Installation Recommendations {#SQLServerdatabases-Tentacles}
+
+Deploying an IIS Web Application or a Windows Service is very straight-forward.  Install the tentacle on the server to be deployed to.  SQL Server is different.  Production SQL Servers are typically clusters or high-availability groups.  They comprise more than one node hidden behind a VIP or virtual IP Address. 
+
+![](common-database-with-vip.png "width=500")
+
+For high-availability groups, there is an active node and a passive node.  In this case, installing a tentacle on each node will not work.  Octopus Deploy will see multiple tentacles and attempt to deploy to both nodes.
+
+SQL PaaS, such as [AWS RDS](https://aws.amazon.com/rds/) or [Azure's SQL](https://azure.microsoft.com/en-us/services/sql-database/), will not allow the installation of tentacles on SQL Server.  
+
+All the tools mentioned above connect to SQL Server using port 1433 and run one or more scripts.  They do not need to be installed directly on SQL Server.  Nor do they need to be run directly on SQL Server.  They will work as long as they run on any machine with a direct connection and port 1433 open.
+
+Also, windows authentication is the often the preferred way to authenticate.  A recommended security practice is the principle of least privilege.  The account used by the website to connect to SQL Server should have little permissions.  Whereas, the account used to make schema changes needs elevated permissions.
+
+Most of the tooling from above requires it to be installed somewhere.  It is important the same version is used across all environments.  This prevents any uncertainty during deployments.  
+
+Finally, it is good security practice to have a different account with schema change permissions per environment.  An account used to change a test environment should not be able to change production.  
```

Comment:
It took me a little while to properly parse: 
Finally, it is good security practice to have a different account with schema change permissions per environment.
Perhaps something like:
Finally, it is good security practice to use different accounts (with schema change permissions) per environment.